utils/spdx/src/main/kotlin/SpdxExpression.kt

```diff
@@ -224,14 +224,14 @@ class SpdxCompoundExpression(
      * Create a compound expression with the provided [operator] and the [left] and [right] child expressions.
      */
     constructor(left: SpdxExpression, operator: SpdxOperator, right: SpdxExpression) :
-        this(operator, listOf(left, right))
+        this(operator, setOf(left, right))
```

Comment:
This could run into an exception as `SpdxCompoundExpression` requires that `children.size > 1`. So basically this change now forbids to make compound expressions with equals operands.

Reply:
I'll simply drop this commit to address this and the above issue.